tv/src/main/java/io/filmtime/tv/ui/home/HomeScreen.kt

```diff
@@ -23,24 +23,25 @@ import io.filmtime.tv.ui.component.MoviesRow
 
 @Composable
 fun HomeScreen(
-  onTopBarVisibleChange: (Boolean) -> Unit,
+  onFirstItemVisibleChange: (isVisible: Boolean) -> Unit,
+  onNavigateToMovieDetail: (tmdbId: Int) -> Unit,
```

Comment:
💅💅💅 Why not `onMovieDetailClick`? maybe parent Composable does not want to navigate and only wants to show a message 😄